docs/guides/cert-manager.md

```diff
@@ -163,12 +163,14 @@ metadata:
   namespace: cert-manager
 spec:
   acme:
-    email: user@example.com # please change this
-    http01: {}
+    email: user@example.com #please change this
     privateKeySecretRef:
       name: letsencrypt-prod
-    server: https://acme-v02.api.letsencrypt.org/directory" | kubectl apply -f -
-clusterissuer.certmanager.k8s.io/letsencrypt-prod created
+    server: https://acme-v02.api.letsencrypt.org/directory
+    solvers:
+    - http01:
+        ingress: {}" | kubectl apply -f -
```

Comment:
I thought I had left a comment here but something is amiss.

My question here: This is empty, how does make a difference?

Reply:
Mostly I converted the format of the ClusterIssuer to match the new required format. I imagine that removing `http01: {}` would fix it as well, but at that point I wasn't sure how everything worked so I found an example that worked and aligned this example with it (since the example also uses an empty field.

Reply:
Question: It is likely that you also had to change the version on Line 159, did you do that change?

Reply:
Nope, there was no version change that I can find after the previous upgrade from cert-manager.io/v1alpha1 -> cert-manager.io/v1alpha2.

Reply:
Hmm, weird. I thought the maintainers did the change in `v1beta1`. I'll merge this in for now and we can change this again if it creates a problem.

Reply:
Unsure. Sounds good!